ddl/constant.go

```diff
@@ -34,11 +38,41 @@ const (
 	HistoryTableID = meta.MaxInt48 - 3
 	// MDLTableID is the table ID of `tidb_mdl_info`.
 	MDLTableID = meta.MaxInt48 - 4
+	// BackfillTableID is the table ID of `tidb_ddl_backfill`.
+	BackfillTableID = meta.MaxInt48 - 5
+	// BackfillHistoryTableID is the table ID of `tidb_ddl_backfill_history`.
+	BackfillHistoryTableID = meta.MaxInt48 - 6
 
 	// JobTableSQL is the CREATE TABLE SQL of `tidb_ddl_job`.
 	JobTableSQL = "create table " + JobTable + "(job_id bigint not null, reorg int, schema_ids text(65535), table_ids text(65535), job_meta longblob, type int, processing int, primary key(job_id))"
 	// ReorgTableSQL is the CREATE TABLE SQL of `tidb_ddl_reorg`.
 	ReorgTableSQL = "create table " + ReorgTable + "(job_id bigint not null, ele_id bigint, ele_type blob, start_key blob, end_key blob, physical_id bigint, reorg_meta longblob, unique key(job_id, ele_id, ele_type(20)))"
 	// HistoryTableSQL is the CREATE TABLE SQL of `tidb_ddl_history`.
 	HistoryTableSQL = "create table " + HistoryTable + "(job_id bigint not null, job_meta longblob, db_name char(64), table_name char(64), schema_ids text(65535), table_ids text(65535), create_time datetime, primary key(job_id))"
+	// BackfillTableSQL is the CREATE TABLE SQL of `tidb_ddl_backfill`.
+	BackfillTableSQL = "create table " + BackfillTable + `(
+		section_id bigint not null,
```

Comment:
It is better not to introduce the concept "section".

How about rename to "id" and rename the "job_id" to "ddl_job_id"?